python/py-ansible-core/Portfile

```
@@ -5,7 +5,7 @@ PortGroup           python 1.0
 PortGroup           select 1.0
 
 name                py-ansible-core
-version             2.11.6
+version             2.12.1
```

It would be great to add a revision line in the ports, for example:

```suggestion
version             2.12.1
revision            0
```